### PR TITLE
feat: diagnose docs

### PR DESCRIPTION
**Pull request by ammarkarachi**
_Issue #, if available:_

_Description of changes:_

Amplify diagnose docs

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by siegerts — APPROVED** [review]

File: src/pages/cli/reference/diagnose.mdx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+export const meta = {
+  title: `Diagnose`,
+  description: `More information about feature flags in Amplify CLI`,
```
Comment (jhockett) [inline src/pages/cli/reference/diagnose.mdx:3]: Should description say feature flags or something like `More information about diagnose command in Amplify CLI`?

File: src/pages/cli/reference/diagnose.mdx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+};
+
+Non-sensitive configurations of your Amplify backend can be securely shared with Amplify by running the command
+
+```bash
+amplify diagnose --send-report
+```
+
+The CLI collects non-sensitive files of your Amplify backend and zips them and transmits the zip file to a secure location. Below are the files CLI collects, transmits and stores.
```
Comment (jhockett) [inline src/pages/cli/reference/diagnose.mdx:12]:
```suggestion
The CLI collects non-sensitive files from your Amplify backend into a zip file and transmits it to a secure location. Below are the files CLI collects, transmits and stores:
```

File: src/pages/cli/reference/diagnose.mdx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+After the zip file is transmitted successfully the current project’s unique identifier is printed. The identifier is used to access the zip file for debugging purposes by the support engineer.
+
+
+```bash
+Project Identifier: 56b5981ed6cf5caad90fb2f8aed150e2
+```
+
+### Automatic Report Sharing
+
+To help improve Amplify CLI you can opt in to automatically share your project configurations with Amplify CLI on failures. The is a project level setting and can be toggled per project. The project can be opt out by running
```
Comment (jhockett) [inline src/pages/cli/reference/diagnose.mdx:32]:
```suggestion

To help improve Amplify CLI you can opt in to automatically share your project configurations with Amplify CLI on failures. The is a project level setting and can be toggled per project. The project can opt out by running
```

File: src/pages/cli/reference/diagnose.mdx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+amplify diagnose --auto-send-off
+```
+
+and opt in by 
+
+```bash
+amplify diagnose --auto-send-on
+``` 
+
+The CLI collects two extra files including the files the configuration files collected in send report
```
Comment (jhockett) [inline src/pages/cli/reference/diagnose.mdx:45]: Can we rewrite this sentence to be easier to understand?

Comment (ammarkarachi) [reply src/pages/cli/reference/diagnose.mdx:45]: Yeah

**Review by jhockett — APPROVED** [review]

**Pull request merged**